test/openapi.yaml

```diff
@@ -163,7 +163,7 @@ paths:
   /parameters/header:
     get:
       parameters:
-        - name: x-param
+        - name: X-Param
```

Comment:
I think you should revert this change and add an explicit test for checking that headers are handled case insensitive.

Reply:
Makes sense. I'll try to get that done today.